src/graphics/scene.odin

```diff
@@ -200,6 +210,8 @@ scene_setup_lights :: proc(frame_count: int) -> (lights: LightData) {
         direction = vec4 { 0, -1, 0, 1 },
         color = vec4 { 1, 1, 1, 1 },
     }
+    lights.data = make([]Light, 1)
```

Comment:
The lights.data field and camera.data field shouldn't be used, I forgot to delete them before and am working on it. Write to the lights.buffer field directly